src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1438,11 +1445,19 @@
         },
 
         "defaultLevel": {
-          "description": "Specifies the default severity level of the result.",
+          "description": "Specifies the default severity level for results generated by this rule.",
           "default": "warning",
           "enum": [ "note", "warning", "error" ]
         },
 
+        "defaultRank": {
```

Comment:
>defaultRank [](start = 9, length = 11)

my apologies. I have, yet again, completely misunderstood your proposal in context of the TC. I never thought we were applying a defaultRank to a rule. This doesn't make sense: ranks are generally computed values that are applied to all results. Ranks are generally available or not. 

when you discussed a default rank, I thought you are asking 'what is the default value of result.rank' if it isn't explicit? in this case, i have argued that the default s/be 0. so what I expected in this case was that we would not have a new ruleConfiguration property but we would have another explanatory sentence in the spec.